Frontend/Helpers/TagHelpers/ProjectStatusTagHelper.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Microsoft.AspNetCore.Razor.TagHelpers;
+
+namespace Frontend.Helpers.TagHelpers
+{
+    [HtmlTargetElement("projectstatus")]
+    public class ProjectStatusTagHelper : TagHelper
+    {
+        public string Status { get; set; } = "NOT STARTED";
```

Comment:
Could this not be the enum, rather than the string? In case we change the display names of stuff